### DISCUSSION TITLE
Plant ID

### DISCUSSION THREAD

**Discussion by ianbsuki** [hultenvp/solis-sensor#265] [Q&A]
Add Ginlong Portal v2 credentials 
Hi, I've followed instruction for plant ID and get xxxxxxxxxxxxxxx5038 a 19 digit number, when inserting into the "plant id" field
the last 2 digits are displayed as "00" as in xxxxxxxxxxxxxxx5000

**Comment by hultenvp** [comment]
I'm not sure what you mean.

Do you mean that in the configuration panel, while entering planId, the plantID value is automatically changed? Or do you see a wrong ID in the logging?

Please provide some more details.

**Reply by ianbsuki** [reply]
Do you mean that in the configuration panel, while entering planId, the plantID value is automatically changed? Yes.

Using the v2 portal. When entering the plant id, the last 2 digits become zero's. If I try to enter the correct digits, ie. 38 they change to zero's. The 17th digit acts a bit strange too, ie. if I enter a 1 it displays a 2, enter a 2 it displays a 2, enter a 3 it displays a 2, 4 shows 4, 5 shows a 6, 6 shows a 6, 7 shows a 6, 8 shows a 8, 9 shows a 0 and increments the preceding digit by 1, 0 shows a 0.

**Reply by hultenvp** [reply]
For plantID on v2 portal (m.ginlong.com) you need to go to the plant info tab and there is a plant ID of 6 long. That one will work for your case.
